x-pack/test/functional/services/infra_source_configuration_flyout.ts

```diff
@@ -4,6 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { Key } from 'selenium-webdriver';
```

Comment:
In order to not break the abstraction introduced by the testing framework we should probably avoid importing from `selenium-webdriver` directly.

How about we access the `Key` object via the `browser` service at the top of the provider function below?

```typescript
  const browser = getService('browser');
```